### PR TITLE
Multiple attributes validates#244

### PR DESCRIPTION
**Pull request by jonnatas**
isuue #244

### CODE REVIEW

**Comment by jonnatas** [comment]
From what I understood, is that? @rafamanzo

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The version numbers below try to follow the conventions at http://semver.org/.
 
 ## Unreleased
 
+- Adding validades with presence uniqueness of ids
```
Comment (rafamanzo) [inline CHANGELOG.md]: What do you think about: `Adds validations (presence and uniqueness) of foreign keys`?

File: app/models/project_attributes.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class ProjectAttributes < ActiveRecord::Base
   belongs_to :user
-  validates :project_id, presence: true
+  validates :project_id, presence: true, uniqueness: { scope: :user_id}
```
Comment (rafamanzo) [inline app/models/project_attributes.rb]: Why have you set this scope?

File: app/models/repository_attributes.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 class RepositoryAttributes < ActiveRecord::Base
   belongs_to :user
-  validates :repository_id, presence: true
+  validates :repository_id, presence: true, uniqueness: { scope: :user_id}
```
Comment (rafamanzo) [inline app/models/repository_attributes.rb]: Why have you set this scope?

Comment (jonnatas) [inline app/models/repository_attributes.rb]: the same here!

File: app/models/project_attributes.rb (hunk 2 above)
Comment (diegoamc) [inline app/models/project_attributes.rb]: Thinking about this once more I don't believe we want to allow multiple project owners for now. Allowing that kind of behaviour may lead to expectations about features we do not yet implement (such as adding members to the project).

So maybe it's better to leave the scope out.

File: app/models/repository_attributes.rb (hunk 3 above)
Comment (diegoamc) [inline app/models/repository_attributes.rb]: Thinking about this once more I don't believe we want to allow multiple project owners for now. Allowing that kind of behaviour may lead to expectations about features we do not yet implement (such as adding members to the project).

So maybe it's better to leave the scope out.

Comment (jonnatas) [inline app/models/repository_attributes.rb]: ok 6fb9480

**Pull request merged**